lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -1361,8 +1326,7 @@ portable, allowing them to work inside an already running event loop or start on
   my $ua = Mojo::UserAgent->new;
   my $mojo   = $ua->get_p('https://metacpan.org/search?q=mojo');
   my $minion = $ua->get_p('https://metacpan.org/search?q=minion');
-  Mojo::Promise->all($mojo, $minion)->then(sub {
-    my ($mojo, $minion) = @_;
+  Mojo::Promise->all($mojo, $minion)->then(sub ($mojo, $minion) {
```

Comment:
Same thoughts here about `_p`-suffixed variables, to make clearer the distinction between those holding promises and those holding the eventual results of them

Reply:
Missed this somehow, variables now suffixed with `_promise` as suggested elsewhere.